index.html

```diff
@@ -0,0 +1,516 @@
+                            <p class="card-text">People in areas of Alabama and Georgia are being urged to seek shelter
+                                after a tornado hit the city of Fultondale, causing damage and multiple injuries.</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-lg-5 col-md-7 section2 ">
+                    <div class="card">
+                        <img class="card-img-top" src="./assets/cards.png" alt="Card image">
+                        <div class="card-body">
+                            <h4 class="card-title">Tornado Hits Fultondale, Alerts in Place for Alabama and Georgia</h4>
+                            <p class="card-text">People in areas of Alabama and Georgia are being urged to seek shelter
+                                after a tornado hit the city of Fultondale, causing damage and multiple injuries.</p>
+                        </div>
+                    </div>
+                    <div class="">
+                        <a href=""><p>MY TURN</p></a>
+                        <div class="row">
+                                    
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <a href=""><img src="assets/image2.jpg" alt="" width="50"></a>
+                                <a href=""><p>'I Married My Best Friend to Avoid Inheritance Tax'</p></a>              
+                            </div>
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <a href=""><img src="assets/image2.jpg" alt="" width="50"></a>
+                                <a href=""><p>'I Married My Best Friend to Avoid Inheritance Tax'</p></a>              
+                            </div>
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <a href=""><img src="assets/image2.jpg" alt="" width="50"></a>
+                                <a href=""><p>'I Married My Best Friend to Avoid Inheritance Tax'</p></a>              
+                            </div>
+                        </div>
+                        <a href=""> <p>CULTURE & TRAVEL</p></a>             
+                        <div class="row">                           
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <a href=""><img src="assets/image2.jpg" alt="" width="50"></a>
+                                <a href=""><p>'I Married My Best Friend to Avoid Inheritance Tax'</p></a>              
+                            </div>     
+                             <div class="col-lg-4 col-md-4 col-sm-12">
+                                <a href=""><img src="assets/image2.jpg" alt="" width="50"></a>
+                                <a href=""><p>'I Married My Best Friend to Avoid Inheritance Tax'</p></a>              
```

Comment:
- This section needs a bit of work.

- [ ]  The links should be not have any text decoration.
- [ ] The image thumbnails are too small

Your clone 👇🏾 
<img width="562" alt="Screen Shot 2021-01-26 at 8 50 51 PM" src="https://user-images.githubusercontent.com/48804258/105897222-2fa8b800-6018-11eb-929a-9e2aa223f5a2.png">

Original page 👇🏾 
![Uploading Screen Shot 2021-01-26 at 8.53.49 PM.png…]()